main.py

```diff
@@ -8,69 +8,98 @@
 import os
 import os.path
 from aiohttp import web
+import json
 
-
-# Customize the cookie name, value and cookie expire date. 
+# Customize the cookie name, value and cookie expire date.
 # The cookie will be stored 30 days long.
 COOKIE_NAME = "OldTamil"
 COOKIE_VALUE = 0
-COOKIE_LIFE_TIME = 24 * 60 * 60 * 30   
-
-
-# User credentials.
-USERNAME = 'Hippod'
-PASSWORD = 'admin'
+COOKIE_LIFE_TIME = 24 * 60 * 60 * 30
 
 # Define the required host and port.
 HOST = '127.0.0.1'
 PORT = 8888
 
 
+# Check for the requested file in the system, return server error! if file not found.
+def check_file(filename):
+    if not os.path.isfile(filename):
+        print("Internal server error! file not found: {}".format(filename))
+        return False
+    response = open(filename).read()
+    return response
+
+
+# User credentials stored in a json file. Loads the json file to access the values.
+# Check if json file can be accessible.
+check = check_file('configuration.json')
+if check:
+    users = open('configuration.json')
+    load_data = json.load(users)
+
+
 def check_cookie(request):
     """cookie_check!
     This function will check for valid cookie on all pages!
     Return True if there is valid cookie.
-
     """
     cookie = request.cookies.get(COOKIE_NAME, None)
-    if not cookie:
-        return False
-    if COOKIE_VALUE == 0:
+    # If there is a cookie, return True.
+    if cookie:
         return True
+    return False
 
 
+# Get the user credentials from json data and validate the user inputs.
 def check_credentials(username, password):
-    if username == USERNAME and password == PASSWORD:
+    if username == load_data['USERNAME'] and password == load_data['PASSWORD']:
```

Comment:
check if a param exist before accessing values! can you be sure that load_data really exists here?!